modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -75,6 +80,10 @@ These parameters are configured in menu:Systems[Bootstrapping].
 You can also configure persistent parameters that are used system-wide, including the sudo user to access the system as an unprivileged user instead of as root.
 
 
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```

Comment:
Same as above. The sudo configuration is not on the container.
We need to explain which steps need to run inside the container and which outside or on a complete different host

Reply:
yes, but within the procedure, we clarify it step-wise ("On each client system" vs. "On the {productname} Server" (now I use "Inside the {productname} Server container, ...").  But I agree that this still can confuse users.